docs/syntax.rst

```diff
@@ -147,36 +166,83 @@ value types. Use literal notation in ``stenotype`` mode.
 
 Set_:
 Set notation is identical to list notation, the difference between them is not
-relevant for annotation. Use literal notation in ``stenotype`` mode.
+relevant for the annotation.
 
 .. code-block:: python
 
+  # value
+  foo = {1, 2, 3, 5, 6, 4}
+
   # regular
   foo: typing.Set[int]
 
   # stenotype
   foo: "{int}"
 
-----
+Signatures
+''''''''''
+
+Specifying the signature of functions is important for wrappers (e.g. decorators),
+callbacks and higher-order functions. It also allows annotating functions from
+third-party libraries which lack annotations.
 
 Callable_:
 In situations where it's not possible to annotate a function in its signature,
 its name can be accessed at a later point in time to add type info.
+This can also be used to specify the type of callbacks or higher-order functions.
 
 .. code-block:: python
 
+  # value
+  def foo(a: str, b: int) -> int:
+      ...
+
   # regular
-  foo: typing.Callable[[str], int]
+  foo: typing.Callable[[str, int], int]
 
   # stenotype
-  foo: "(str) -> int"
+  foo: "(str, int) -> int"
 
-----
+.. note::
```

Comment:
the note block doesn't render correctly in the github rst view, so I don't know if it looks good. Could you rebase on master or merge it back in? I changed the CI to run on PRs so that I can take a look at reference artifacts.